pkg/resources/cloudconfig/configmap.go

```diff
@@ -144,6 +145,7 @@ func CloudConfig(
 			},
 			LoadBalancer: openstack.LoadBalancerOpts{
 				ManageSecurityGroups: manageSecurityGroups == nil || *manageSecurityGroups,
+				UseOctavia: useOctavia == nil || *useOctavia,
```

Comment:
I would also add a test case in `pkg/resources/cloudconfig/configmap_test.go`.

Reply:
Afaik, there is only `TestGetVsphereCloudConfig` and for Openstack it would require a complete new Test. This is somehow out of scope for this tiny change, isn't it?
But, I agree there should be a `TestGetOpenstackCloudConfig`!

Reply:
Yep, you are right. What I would recommend is to:
* Renaming the test to `TestCloudConfig`
* Call the function `CloudConfig` instead of `getVsphereCloudConfig`.
* Add the testcase for openstack.
It should not be too complicated unless I missed something, it would be great if you can give it a try!

Reply:
About this change, this requires a bit more effort I guess.
The CloudConfig function returns a string what is really helpful. I would move this to a new issue where it makes sense to have CloudConfig using individual `getXCloudConfig` functions. e.g. `getOpenstackCloudConfig`. This allows to simplify the CloudConfig function and gives the flexibility to use the CloudConfig from other providers within `configmap_test.go`.
What do you think?

Reply:
Yes, what I would do is to have the test verifying the marshaled version of the cloud-config instead, I agree this requires some more effort. I agree this can be tackled in a follow-up PR.